wsi/drm/drmdisplay.cpp

```diff
@@ -393,7 +391,9 @@ void DrmDisplay::UpdateDisplayConfig() {
     return;
   }
   flags_ |= DRM_MODE_ATOMIC_ALLOW_MODESET;
-  SetDisplayAttribute(modes_[config_]);
+  uint32_t activeconfig = 0;
+  PhysicalDisplay::GetActiveConfig(&activeconfig);
```

Comment:
this line is just to get activeconfig=config_?

Reply:
config_ is moved into private in PhysicalDisplay